controlplane/kubeadm/internal/webhooks/kubeadm_control_plane.go

```diff
@@ -455,6 +462,37 @@ func validateRolloutStrategy(rolloutStrategy *controlplanev1.RolloutStrategy, re
 	return allErrs
 }
 
+func validateNamingStrategy(machineNamingStrategy *controlplanev1.MachineNamingStrategy, pathPrefix *field.Path) field.ErrorList {
+	var allErrs field.ErrorList
+
+	if machineNamingStrategy.Template != "" {
+		if !strings.Contains(machineNamingStrategy.Template, ".random") {
+			allErrs = append(allErrs,
+				field.Invalid(
+					pathPrefix.Child("template"),
+					machineNamingStrategy.Template,
+					"invalid ControlPlane name template, {{ .random }} variable is missed",
```

Comment:
```suggestion
					"invalid template, {{ .random }} is missing",",
```
The error will contain the entire field path, so we can keep the error a bit more concise